Dockerfile

```diff
@@ -43,4 +43,5 @@ WORKDIR /build
 
 RUN composer install && \
     vendor/bin/phpcs --standard=./phpcs-ruleset.xml && \
-    vendor/bin/phpunit
+    vendor/bin/phpunit && \
+    php -d extension=opencensus.so vendor/bin/phpunit
```

Comment:
The Dockerfile builds the extension and installs it before running the php tests. The Dockerfile is parameterized by the base image so it runs against php 7.1, 7.0, 5.6 as well as 7.1 and 7.0 for 32-bit architecture. The `make install` puts the .so file in the php extension load path, but does not enable it.